app/helpers/utils.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import logging.config
+import os
+from itertools import chain
+
+import yaml
+
+logger = logging.getLogger(__name__)
+
+
+def get_logging_cfg():
+    cfg_file = os.getenv('LOGGING_CFG', 'logging-cfg-local.yaml')
+
+    config = {}
+    with open(cfg_file, 'rt') as fd:
```

Comment:
with the current pylint version, this will cause a warning (opening without specifying an encoding). I know from my very recent experience ;-)